marklogic-data-hub/src/test/java/com/marklogic/hub/flow/FlowRunnerTest.java

```diff
@@ -133,7 +140,16 @@ public void testIngestCSVasXML(){
         EvalResult res = resultItr.next();
         long count = Math.toIntExact((long) res.getNumber());
         Assertions.assertEquals(count, 25);
-    }
+        if(! isCertAuth() ) {
+           EvalResultIterator itr = client.newServerEval().xquery("xdmp:estimate(fn:collection('http://marklogic.com/provenance-services/record'))").eval();
+           if(itr != null && itr.hasNext()) {
+               Assertions.assertTrue(itr.next().getNumber().intValue() == 25);
```

Comment:
It's better to use assertEquals with an expected value of 25 - that way if the test fails, the fail message states the expected and actual values instead of just saying "I expected true but it was false".